convert2rhel/actions/system_checks/check_firewalld_availability.py

```diff
@@ -0,0 +1,109 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+import os
+
+from convert2rhel import actions, systeminfo
+from convert2rhel.systeminfo import system_info
+
+
+logger = logging.getLogger(__name__)
+
+FIREWALLD_CONFIG_FILE = "/etc/firewalld/firewalld.conf"
+
+
+def _check_for_modules_cleanup_config():
```

Comment:
The function returns boolean so better to name it as a question.
```suggestion
def _is_modules_cleanup_enabled():
```